src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/OsgiFramework.kt

```diff
@@ -135,7 +135,7 @@ class OsgiFramework(sync: InstanceSync) : InstanceService(sync) {
 
     fun findConfiguration(pid: String): Configuration? {
         return try {
-            sync.http.get("$CONFIGURATION_PATH/$pid?post=true") { asObjectFromJson(it, Configuration::class.java)  }
+            sync.http.get("$CONFIGURATION_PATH/$pid?post=true") { asObjectFromJson(it, Configuration::class.java) }
```

Comment:
why GET request has `post=true` param? is it really needed?

Reply:
This way we can get an OSGi configuration in `json` format without unnecessary wrappers

Reply:
ok